docs/docs/build/dashboards/customization.md

```diff
@@ -13,45 +13,17 @@ For a full list of available dashboard properties and configurations, please see
 
 :::
 
+## Define Dashboard Access
```

Comment:
It feels like dashboard-specific security policies should be a separate page. What do you think?